packages/backend/src/config/app.ts

```diff
@@ -145,17 +143,22 @@ function parseRedisTlsConfig(
 }
 
 // exported for testing
-export function parseOrProvisionKey(keyPath: string): crypto.KeyObject {
-  try {
-    const key = crypto.createPrivateKey(fs.readFileSync(keyPath))
-    const jwk = key.export({ format: 'jwk' })
-    if (jwk.crv === 'Ed25519') {
-      return key
-    } else {
-      console.log('Private key is not EdDSA-Ed25519 key. Generating new key.')
+export function parseOrProvisionKey(
+  keyFile: string | undefined
+): crypto.KeyObject {
+  if (keyFile) {
+    try {
+      const key = crypto.createPrivateKey(fs.readFileSync(keyFile))
+      const jwk = key.export({ format: 'jwk' })
+      if (jwk.crv === 'Ed25519') {
```

Comment:
It would be nice to have `'Ed25519'` coming from a type declaration, but it looks like we are limited to just a string here.

Reply:
Maybe we could eventually do schema validation using:
https://github.com/interledger/open-payments/blob/b125873edb6d33acd40eb304e026e5111ec1dba9/openapi/shared/schemas.yaml#L201-L234